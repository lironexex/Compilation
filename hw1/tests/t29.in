// comments 9 - nested comments 2 - a multi-line comment nested inside a multi-line comment is illegal:
/* outer comment: 
some stuff here
// this is legal
"were are still ok"
"still fine \u{412sdfsd}"
/* illegal nested comment */ */ 
