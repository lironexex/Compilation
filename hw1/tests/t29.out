1 COMMENT 1
Warning nested comment
